### ISSUE TITLE
Running Graphitti for the commit ID also outputs extra stuff

### ISSUE THREAD

**Issue by stiber** [UWB-Biocomputing/Graphitti#290]
While Workbench could certainly grep for the ID, running `cgraphitti -v` or `ggraphitti -v` should just output the commit ID and nothing else.

**Comment by VivekGandhi54** [comment]
I had asked you about this before, but we decided to implement it in `parseCommandLine()` using `ParamContainer`, which is only called after other stuff is printed. I can change it so that the -v arg is checked at the start of `main`.

**Comment by stiber** [comment]
Good point. Maybe we can look into parsing tinted commands line early in the overall control flow.
